plugins/dynamic-plugins-info/src/plugin.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  createApiFactory,
+  createPlugin,
+  createRoutableExtension,
+  discoveryApiRef,
+  fetchApiRef,
+} from '@backstage/core-plugin-api';
+
+import { DynamicPluginsInfoClient } from './api/DynamicPluginsInfoClient';
+import { dynamicPluginsInfoApiRef } from './api/types';
+import { dynamicPluginsInfoRouteRef } from './routes';
+
+export const dynamicPluginsInfoPlugin = createPlugin({
+  id: 'dynamic-plugins-info',
+  routes: {
+    root: dynamicPluginsInfoRouteRef,
+  },
+  apis: [
+    createApiFactory({
+      api: dynamicPluginsInfoApiRef,
+      deps: {
+        discoveryApi: discoveryApiRef,
+        fetchApi: fetchApiRef,
+      },
+      factory: ({ discoveryApi, fetchApi }) =>
+        new DynamicPluginsInfoClient({ discoveryApi, fetchApi }),
+    }),
+  ],
+});
+
+export const DynamicPluginsInfoTab = dynamicPluginsInfoPlugin.provide(
```

Comment:
I chose this name for that reason actually, as this later on becomes the name you would provide in the configuration.  Introducing another name seemed like it would be confusing to the user, but is there some potential issue I could be introducing following this approach?

Reply:
no issues as such and is just nit, as felt if we can different name would not be confusing,

Reply:
Yeah, thinking about it further I haven't picked such a great name anyways since this isn't the tab, it's the tab content.  I'll change this export to `DynamicPluginsInfo` and rename the component to `DynamicPluginsInfoContent` to distinguish between these but maintain the correlation.